staging_service/app.py

```
@@ -640,6 +654,11 @@ def inject_config_dependencies(config):
     if Path._CONCIERGE_PATH is None:
         raise Exception("Please provide CONCIERGE_PATH in the config file ")
 
+    if Path._DTS_MANIFEST_SCHEMA_PATH is None:
+        raise Exception("Please provide DTS_MANIFEST_SCHEMA in the config file")
+    global _DTS_MANIFEST_SCHEMA
+    _DTS_MANIFEST_SCHEMA = DTS_MANIFEST_SCHEMA_PATH
```

Shouldn't you load / parse the schema here to fail early?

---

Can you add a comment to the config / schema parsing code noting that there's no tests, it's all tested manually, and if you make changes you need to add tests or test manually?

---

Noted.

---

: